### PR TITLE
Loadout guns! pew pew

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Adds in the .35 op havelock to the loadout, starts unloaded so you need 3 points to even get it loaded on spawn
Adds in a few rubber and mags for .35 to loadout
Adds in a pre-loaded Clarissa  with rubbers
Adds in for 1 costs gun case to loadout

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	path = /obj/item/ammo_magazine/speed_loader_pistol_35/rubber
+	cost = 1
+
+/datum/gear/armament/ammo_box
+	display_name = "Rubber .35 Auto pistol box"
+	path = /obj/item/ammo_magazine/ammobox/pistol_35/rubber
+	cost = 2
+
+/datum/gear/armament/detective
+	display_name = "Enemty Havelock .35 Auto pistol"
```
Comment (TheSleepyCatmom) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:31]:
```suggestion
	display_name = "Unloaded Havelock .35 Auto pistol"
```

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	path = /obj/item/ammo_magazine/highcap_pistol_35/rubber
+	cost = 2
+
+/datum/gear/armament/ammo_speed
+	display_name = "Rubber .35 Auto pistol speed loader"
+	path = /obj/item/ammo_magazine/speed_loader_pistol_35/rubber
+	cost = 1
+
+/datum/gear/armament/ammo_box
+	display_name = "Rubber .35 Auto pistol box"
```
Comment (TheSleepyCatmom) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:26]:
```suggestion
	display_name = "Rubber .35 Auto pistol ammo box"
```

**Review by TheSleepyCatmom — COMMENTED** [review]
<3 Just some suggestions to make it read better and be more clear (plus fixing your silly spelling, catto.)

**Comment by cdb-is-not-good** [comment]
good pr overall- though i dont really agree with making /all/ of them spawn with rubbers since rubbers are fairly...meh, across the board.

**Pull request merged**